cmd/tailscale/cli/debug.go

```diff
@@ -724,7 +724,11 @@ var debugComponentLogsArgs struct {
 
 func runDebugComponentLogs(ctx context.Context, args []string) error {
 	if len(args) != 1 {
-		return errors.New("usage: debug component-logs <component>")
+		errStr := "usage: debug component-logs <component>\n\n<component> may be: \n\n"
```

Comment:
```suggestion
		errStr := "usage: tailscale debug component-logs <component>\n\n<component> may be: \n\n"
```

might be worth including the root command here, too. If not "tailscale" then `filepath.Base(os.Args[0])`.